### PR TITLE
Cans 213 - Replace wood duck global page headers

### PR DESCRIPTION
**Pull request by abelayt**
## Tests

- [ ] I used TDD to develop the feature
- [ ] I have included unit tests
- [ ] I have included new acceptance tests or modified existing ones
- [ ] I have included other tests

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [ x] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x ] My code follows the code style of this project.
- [x ] I ran all my tests locally which were green.
- [ x] My code adds no new issues to Code Climate
- [x ] I ran all the linters for the project.
- [x ] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build (or bring donuts if I leave things broken), to check linters, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

**Comment by abelayt** [comment]
As agreed during the standup this morning, only the Page Header is replaced and still using the Global Header from react-wood-duck.

File: client/src/components/Header/Header.test.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ describe('Header', () => {
       const wrapper = await mount(<Header />);
       const displayedUserInitials = wrapper
         .find('.profile-avatar')
-        .text()
+      .text()
```
Comment (denys-davydov) [inline client/src/components/Header/Header.test.js]: This formatting is inconsistent, need to revert it.

File: client/src/components/Layout/PageInfo.test.js [hunk 2]
```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import { shallow } from 'enzyme';
-import { Row, Col, Breadcrumb, BreadcrumbItem } from 'reactstrap';
+// import { Row, Col, Breadcrumb, BreadcrumbItem } from 'reactstrap';
```
Comment (denys-davydov) [inline client/src/components/Layout/PageInfo.test.js]: Looks like you've forgotten to delete this line

File: client/src/components/Header/Header.test.js (hunk 1 above)
Comment (abelayt) [reply client/src/components/Header/Header.test.js]: fixed.

File: client/src/components/Layout/PageInfo.test.js (hunk 2 above)
Comment (abelayt) [reply client/src/components/Layout/PageInfo.test.js]: comment removed.

**Pull request merged**